tests/test_sklearn_transform.py

```diff
@@ -0,0 +1,48 @@
+import re
+
+import pytest
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.feature_selection import SelectPercentile, SelectKBest
+from sklearn.pipeline import FeatureUnion
+from sklearn.datasets import load_iris
+from eli5 import transform_feature_names
+
+iris_X, iris_y = load_iris(return_X_y=True)
```

Comment:
Could you please change it to use a pytest fixture? There is already a fixture for iris (https://github.com/TeamHG-Memex/eli5/blob/master/tests/conftest.py#L61); using it will ensure iris is not loaded on test discovery, and that it only loads once. This is not very important for iris, but it is nice to have code consistent. To use the fixture add 'iris_train' argument to test_transform_feature_names_iris function; a tuple (X, y, feature_names, target_names) will be passed to the test function in this argument.